pages/dkp/kommander/2.2/clusters/creating-konvoy-cluster-aws/index.md

```diff
@@ -22,8 +22,10 @@ From the **Add Cluster** menu, select the **Create Konvoy Cluster** and provide
 
 Select the pre-configured [AWS infrastructure provider](../../operations/infrastructure-providers/configure-aws-infrastructure-provider-static-credentials/) (or [AWS role credentials](../../operations/infrastructure-providers/configure-aws-infrastructure-provider-roles/)) to display the remaining options specific to AWS.
 
-- **Region**: Select the data center region to reveal selectable Availability Zones.
+- **Region**: Select the data center region or specify a custom region.
 - **Add Infrastructure Provider Tags**: Specify tags to be set on all resources created in your infrastructure provider for this cluster. You can add up to 10 tags in this form. Different infrastructure providers have varying restrictions on their tags. See the [AWS Tags User Guide][aws-tags] for more information on using tags in AWS.
+- **Configure Node Pools**: Specify pools of nodes, their size, and quantity. The default suggests a control plane pool containing 3 m5.xlarge nodes and a worker pool containing 4 m5.2xlarge nodes. You can also specify IAM instance profile, base OS, lookup format, owner ID, and AMI ID as part of each pool.
```

Comment:
I would remove the default suggestions here as we would need to update the docs if we would update it in the code

Reply:
```suggestion
- **Configure Node Pools**: Specify pools of nodes, their machine types, and quantity. You can also specify the IAM instance profile, base OS, lookup format, owner ID, and AMI ID as part of each pool.
```

Reply:
We might want to make an indented list here so we can have a line for each item. Then we can give details for the specific option.

I think we should also mention that the control-plane size has to be odd. 

The base OS, lookup format, owner ID, and AMI ID can be grouped together as they all pertain to the AMI that the nodes will use. To be accurate the options are base OS, lookup format, owner ID, **or** AMI ID. base OS, lookup format, owner ID are used to lookup the AMI, or you can specify a specific AMI with the AMI ID. If you set AMI ID we disable base OS, lookup format, owner ID in the UI.

Reply:
Yeah to Rodney's point we need to make sure that you can input AMI ID OR base OS, lookup format, owner ID. Not that the software will let you.